src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -235,14 +261,29 @@ public string ToString(string directory)
             int displayLineNumber = this.LineNumber - Context.DisplayPreContext.Length;
             foreach (string contextLine in Context.DisplayPreContext)
             {
-                lines.Add(FormatLine(contextLine, displayLineNumber++, displayPath, ContextPrefix));
+                
+                if(!isRaw){
+                    lines.Add(FormatLine(contextLine, displayLineNumber++, displayPath, ContextPrefix));
+                }else{
+                    lines.Add(contextLine);
+                }
             }
 
-            lines.Add(FormatLine(Line, displayLineNumber++, displayPath, MatchPrefix));
-
+            if(!isRaw){
```

Comment:
Same as above on the positive case first

```suggestion
            if (isRaw)
            {
```